### PR TITLE
Option to use RVM for ruby

### PR DESCRIPTION
**Pull request by jjrh**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,3 +1,4 @@
+SHELL := /bin/bash --login
```
Comment (jcbrand) [inline Makefile]: Looks like this variable isn't being used. Why did you add it?

Comment (jjrh) [reply Makefile]: I believe that is used when commands are run - otherwise, at least on my machine sh is used which causes some issues mostly with RVM.

Comment (jjrh) [reply Makefile]: And yes, RVM is like virtualenv. On my machine the system version of ruby is pretty ancient so some things break.
